src/main/java/com/google/sps/data/Note.java

```diff
@@ -107,18 +108,16 @@ public String getFileName(){
     }
   }
 
-  public void writeConvertedDoc() throws Docx4JException, IOException {
+  public void writeConvertedDoc(OutputStream outStream) throws Docx4JException, IOException {
     //if no categories classified then the title is "Getcha Notes" by default
     String title = !categories.isEmpty() ? categories.get(0) : "Getcha Notes";
 
     WordprocessingMLPackage wordMLPackage = WordprocessingMLPackage.createPackage();
     wordMLPackage.getMainDocumentPart().addStyledParagraphOfText("Title", title);
     wordMLPackage.getMainDocumentPart().addParagraphOfText(message);
 
-    // saved in Team-33-Note-App/target/portfolio-1
     fileName = String.format("Getcha_Notes_%d", sessionKey.getId());
-    filePath = String.format("%s/%s", System.getProperty("user.dir"), fileName);
```

Comment:
I think you might be able to get rid of the `fileName` and `filePath` variables now? Are they used anywhere?

Reply:
`fileName` is still used to assign a unique file name for the Note object based on the session ID